### PR TITLE
[serverless-1.34] More granular image references

### PR DESCRIPTION
**Pull request by matejvasek**
More granular image references.

### CODE REVIEW

**Comment by matejvasek** [comment]
PTAL @jrangelramos @dsimansk

File: pkg/pipelines/tekton/tasks.go [hunk 1]
```diff
@@ -5,7 +5,24 @@ import (
 	"strings"
 )
 
-var DeployerImage = "ghcr.io/knative/func-utils:latest"
+var (
+	FuncUtilImage = "ghcr.io/knative/func-utils:latest"
+	DeployerImage string
+	ScaffoldImage string
+	S2IImage      string
```
Comment (dsimansk) [inline pkg/pipelines/tekton/tasks.go:12]: Expected to be set through `ldflags`?

Comment (matejvasek) [reply pkg/pipelines/tekton/tasks.go:12]: yep they could be overriden via ldflags if neccessary, if unset the init block below should fall back to the FuncUtilImage.

**Pull request merged**